### DISCUSSION TITLE
7.99版本经常卡在加载界面

### DISCUSSION THREAD

**Discussion by laowang96** [getlantern/lantern-forum-cn#1078] [Q&A] [answered]
支持报告问题也弄不了，提示出错，希望能及时更新解决。并且付费版反而没有免费的快怎么回事

**Comment by MaxMKMax** [comment, marked as answer]
@laowang96 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by laowang96** [reply]
就是app报告功能不能用，提示我们遇到了技术问题，稍候再试

**Reply by laowang96** [reply]
![Screenshot_2024-12-11-09-15-57-42_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/user-attachments/assets/31a84745-0a94-48d1-81ca-1ac5f697db62)

**Comment by laowang96** [comment]
😑

**Comment by MaxMKMax** [comment]
@laowang96 请问您是用哪个系统的电脑还是手机无法提交报告? IOS? Android? Mac OS? Window?
我帮您强力反馈上去!!!

**Reply by laowang96** [reply]
Android
